iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/match/voter/FittingAffWordsMatchVoter.java

```diff
@@ -0,0 +1,157 @@
+package eu.dnetlib.iis.wf.affmatching.match.voter;
+
+import java.util.List;
+import java.util.Set;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.Sets;
+
+import eu.dnetlib.iis.wf.affmatching.model.AffMatchAffiliation;
+import eu.dnetlib.iis.wf.affmatching.model.AffMatchOrganization;
+import eu.dnetlib.iis.wf.affmatching.orgsection.OrganizationSectionsSplitter;
+
+/**
+ * Match voter that checks if {@link AffMatchAffiliation#getOrganizationName()} section 
+ * words are present in {@link AffMatchOrganization#getName()} words.
+ * 
+ * @author madryk
+ */
+public class FittingAffWordsMatchVoter implements AffOrgMatchVoter {
+
+    private static final long serialVersionUID = 1L;
+    
+    
+    private OrganizationSectionsSplitter organizationSectionsSplitter = new OrganizationSectionsSplitter();
+    
+    private List<Character> charsToFilter;
+    
+    private double minFittingOrgWordsRatio;
+    
+    private double minFittingWordSimilarity;
+    
+    private int minWordLength;
+    
+    
+    //------------------------ CONSTRUCTORS --------------------------
+    
+    /**
+     * Default constructor
+     * 
+     * @param charsToFilter - list of characters that will be filtered out before comparing words
+     * @param minWordLength - words with length equal or less than 
```

Comment:
I would name it wordToRemoveMaxLength because otherwise it is misleading, it can be seen as min length of words that will be retained.